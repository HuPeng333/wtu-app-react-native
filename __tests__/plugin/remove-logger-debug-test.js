const babel = require('@babel/core')
const plugin = require('../../plugin/remove-logger-debug.js')

const example = `
const logger = getLogger('/redux/counter/messageSlice')

const test = () => {
logger.debug('')
logger.info()
logger.debug()
}
`

it('works', () => {
  const { code } = babel.transform(example, {
    plugins: [plugin],
    presets: [],
    filename: '',
  })
  expect(code).toMatchSnapshot()
})
