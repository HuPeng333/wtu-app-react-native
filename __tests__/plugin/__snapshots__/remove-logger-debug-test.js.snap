// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`works 1`] = `"var logger=getLogger('/redux/counter/messageSlice');var test=function test(){logger.info();};"`;
